packages/bpk-component-graphic-promotion/src/__snapshots__/BpkGraphicPromo-test.tsx.snap

```diff
@@ -412,12 +372,14 @@ exports[`BpkGraphicPromo should support custom class names 1`] = `
   <div
     aria-label="Sponsored. Airline Name. Ride your wave. Portugal and 6 more countries have just been added to the UK travel green list. Learn more"
     class="bpk-card bpk-graphic-promo custom-classname"
+    id=""
```

Comment:
Let's update the tests to use the content ID. I would also suggest we do not set the `id` when `contentId` is not there.

Reply:
Continuing in https://github.com/Skyscanner/backpack/pull/2535#discussion_r952735241

Reply:
Add contentId to tests: https://github.com/Skyscanner/backpack/pull/2544